ethereum-explorer/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+mod pb;
+
+#[path = "map_block_meta.rs"]
+mod block_meta;
+
+#[path = "map_filter_transaction.rs"]
+mod filter_transaction;
+
+#[path = "map_contract_events.rs"]
+mod contract_events;
```

Comment:
That is a weird thing, I would try to avoid that. Here an example of "code splitting" I liked: https://github.com/pinax-network/substreams-cookbook/blob/main/erc20/src/lib.rs

Reply:
yeah that's nice!

Reply:
I got the idea from https://github.com/streamingfast/substreams-eth-block-meta/blob/ba2389c31de006b77461aab5e925a6e5177d2a8a/src/lib.rs#L2. Do you want me to update that project too?

Reply:
Yeah I think we should have not used it there indeed (I think I'm the one who did it :D).

For there, we would decide on:
- rename `db_out.rs` to `db.rs` 
- or renamed `mod db` to `mod db_out`.